packages/react-storage/src/components/StorageBrowser/Views/Controls/Paginate.tsx

```diff
@@ -212,11 +224,6 @@ const paginateComponents = {
   Icon: IconElement,
 };
 
-PaginateControl.Container = PaginateContainer;
-PaginateControl.Current = Object.assign({}, CurrentControl, paginateComponents);
-PaginateControl.Next = Object.assign({}, NextControl, paginateComponents);
-PaginateControl.Previous = Object.assign(
-  {},
-  PreviousControl,
-  paginateComponents
-);
+PaginateControl.Current = Object.assign(CurrentControl, paginateComponents);
+PaginateControl.Next = Object.assign(NextControl, paginateComponents);
+PaginateControl.Previous = Object.assign(PreviousControl, paginateComponents);
```

Comment:
Can we remove the `paginateComponents` without issue here?

Reply:
Hmm no, this actually ends up being the same issue I brought up in the Summary PR (which I now see why it wasn't working, but with removing the composability will have the same issue here); it results in:

<img width="874" alt="Screenshot 2024-07-25 at 3 00 32 PM" src="https://github.com/user-attachments/assets/27485734-db3e-4996-81db-8b27c0d8e230">

Reply:
Resolved the above error in latest